tests/test_core.py

```diff
@@ -1915,6 +1945,8 @@ def test_stack_void(self):
 
     self.do_run_from_file(src, output)
 
+  # Broken for binaryen2
```

Comment:
Oh, I was thinking that it was hanging in compile, but you're right, it's at runtime. Also it's not just binaryen2, it's also binaryen0 for the same reason.